Lux/lux_react/src/components/InterfaceContainer.jsx

```diff
@@ -53,8 +61,19 @@ function InterfaceContainer({panelSize}) {
             }
             return newValues;
         });
+        
+        // Notify audio system about manual slider changes
+        if (window.audioHook && window.audioHook.handleManualSliderChange) {
```

Comment:
How does this work? I cannot find `handleManualSliderChange` anywhere else in the code. Perhaps this is not needed - audio shouldn't know anything about sliders, just send its values to the audio functor.